scripts/static-serve.js

```diff
@@ -33,14 +33,6 @@ const argv = yargs
 // Set the path for static file serve
 app.use(argv['base-path'], express.static(path.join(process.cwd(), argv.dist)))
 
-// Setup server shutdown signal from parent
-process.on('SIGTERM', () => {
```

Comment:
not needed anymore?

Reply:
Seems like this event is never triggered when the main process uses `kill()` on the child process. So I  removed it.